### PR TITLE
Initial skeleton for controllers and factories

### PR DESCRIPTION
**Pull request by antonioparisi**

### CODE REVIEW

File: config/routes.rb [hunk 1]
```diff
@@ -1,3 +1,11 @@
 Rails.application.routes.draw do
   devise_for :users
+
+  namespace :api do
+    namespace :v1 do
+      resources :streamings, only: [:index, :create, :show] do
+        resources :comments, only: [:index, :create]
```
Comment (djGrill) [inline config/routes.rb]: I think we should have a `destroy` endpoint for both; as `Streamings` and `Comments` will be deletable

File: spec/factories/users.rb [hunk 2]
```diff
@@ -0,0 +1,12 @@
+FactoryGirl.define do
+  factory :user do
+    sequence(:id) { |id| id }
```
Comment (djGrill) [inline spec/factories/users.rb]: we don't need to set the `id`; as it's set by `ActiveRecord` automatically

File: spec/factories/users.rb [hunk 3]
```diff
@@ -0,0 +1,12 @@
...
+
+    email { Faker::Internet.email }
+    password { 'password' }
+    password_confirmation { 'password' }
+    is_guest false
+
+    platform { 1 }
```
Comment (djGrill) [inline spec/factories/users.rb]: on the specs let's use the string version of the platform so it's easy and clear which one is it (`ios`, `android`)\

also please add the enum to the model

File: spec/factories/streamings.rb [hunk 4]
```diff
@@ -0,0 +1,5 @@
+FactoryGirl.define do
```
Comment (djGrill) [inline spec/factories/streamings.rb]: let's also add a `comments` factory

File: spec/factories/users.rb [hunk 5]
```diff
@@ -0,0 +1,12 @@
+FactoryGirl.define do
+  factory :user do
+    sequence(:id) { |id| id }
+
+    email { Faker::Internet.email }
```
Comment (djGrill) [inline spec/factories/users.rb]: we need to add `faker` to the `Gemfile`

Comment (djGrill) [inline spec/factories/users.rb]: mmm shall we use `faker` or `ffaker`?

Comment (antonioparisi) [inline spec/factories/users.rb]: oh yeah, I did. I forgot to push it. give me a sec

Comment (antonioparisi) [inline spec/factories/users.rb]: `ffaker`? 👍

Comment (djGrill) [inline spec/factories/users.rb]: mmm it doesn't look like anything special. both are being actively mantained :o

**Pull request merged**